### PR TITLE
Add an embed for tito as well as make the ticket button generic for all

### PR DESCRIPTION
**Pull request by ukmadlz**
@jmsv we may want to change the button colour too?

### CODE REVIEW

**Comment by jmsv** [comment]
> @jmsv we may want to change the button colour too?

yeah good call

probably move the Eventbrite/Tito check out of `EventEmbed` and pass it as a prop so the value can also be used for conditional `className` on the button

looks like we want `#0079ff`

**Comment by ukmadlz** [comment]
I've tested it on https://ti.to/hackference/hackference-2018

**Comment by ukmadlz** [comment]
Are you wanting to change the colour depending on the ticket provider?

**Comment by jmsv** [comment]
> Are you wanting to change the colour depending on the ticket provider?

yup - was thinking that's what you meant by "want to change the button colour too"?

**Comment by ukmadlz** [comment]
@jmsv I have 0 idea why but the CSS isn't applying https://fusion-meetup-aqtndzugo-fusion-meetup.vercel.app/events/meetup-2022-09-29

**Comment by jmsv** [comment]
> CSS isn't applying

looks like you're building CSS classes dyanmically so Tailwind & PurgeCSS decides they don't exist

**Comment by ukmadlz** [comment]
> 

Suggestions welcome?

**Comment by jmsv** [comment]
got it working but the blue looks out of place 😛 

image

thinking about it though, if we're going for "tickets" rather than the name of the ticket platform, it doesn't really need to use the ticket brand colour

**Comment by ukmadlz** [comment]
I do think it needs a CTA colour however?

**Comment by jmsv** [comment]
> I do think it needs a CTA colour however?

agree - this good?

image

File: components/events/EventCardSmall.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ export const EventCardSmall: React.FC<EventCardSmallProps> = ({
         </h2>
 
         <div className="pt-2">
-          <EventDetails event={event} small />
+          <EventDetails event={event} small ticketProvider={""} />
```
Comment (jmsv) [inline components/events/EventCardSmall.tsx:34]: could make this an optional prop?

Comment (ukmadlz) [reply components/events/EventCardSmall.tsx:34]: It threw a benny with the provider colour matching object. It would work as an optional now?

Comment (jmsv) [reply components/events/EventCardSmall.tsx:34]: yup should do!

**Review by jmsv — APPROVED** [review]
noice

**Pull request merged**